Missing note about necessary libgeoip-dev dependency on GNU/Linux

Hio, cool project. I was trying to get a local copy up and running and ran into the following:

``` bash
$ make venv
$ pip install -r requirments.txt
... # success!
$ pip install -r dev-requirements.txt
...  # lots of noise
py_GeoIP.c:23:19: fatal error: GeoIP.h: No such file or directory
compilation terminated.
error: command 'gcc' failed with exit status 1
```

This was solved with:

``` bash
    $ sudo apt-get install -y libgeoip-dev
```

I couldn't find it listed in the documentation. 

Where could I add it? I'd submit a PR.

--- a/README.rst
+++ b/README.rst
@@ -172,6 +172,13 @@ Using pip this would be::
     pip install Twisted ipaddr pygeoip
     pip install GeoIP Sphinx repoze.sphinx.autointerface coverage  # for development
 
+.. note::
+    Don't forget to install the GeoIP dev headers if you are developing:
+
+    .. code:: bash
+
+        apt-get install libgeoip-dev
+
 or::
 
     pip install -r requirements.txt

